tests/Cache/HazelCastTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Memcached as Memcached;
+use Utopia\Cache\Cache;
+use Utopia\Cache\Adapter\Hazelcast as HazelCastAdapter;
+use Utopia\Tests\Base;
+
+class HazelCastTest extends Base
+{
+    public static function setUpBeforeClass(): void
+    {
+        $memcached = new Memcached();
+        $memcached->addServer('127.0.0.1', 5701);
```

Comment:
127.0.0.1 is probably not going to work because the test container will try to connect to itself rather than Hazelcast. See what's used in the other tests.